src/CMakeLists.txt

```diff
@@ -23,6 +23,18 @@ add_einsums_library(einsums
         Einsums::h5cpp
 )
 
+if(EINSUMS_ENABLE_HIP)
```

Comment:
This can be changed to:

```
   extend_einsums_target(einsums
     SOURCES
         GPUTensorAlgebra.hip
     DEPENDS
         hip-lang::device
         hip-lang::host
         rccl
     CONDITION
         EINSUMS_ENABLE_HIP
   )
```

Reply:
Did not know about this. I’ll change it